.circleci/config.yml

```diff
@@ -210,6 +210,7 @@ jobs:
     docker:
       - image: cimg/base:2022.08
     steps:
+      - checkout
```

Comment:
Would we also need to add this to publish prod?

Reply:
nope, we don't run any git commands for prod publish.